tests/kola/destructive/var-mount.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# https://github.com/ostreedev/ostree/issues/1667
+set -xeuo pipefail
+
+. ${KOLA_EXT_DATA}/libinsttest.sh
+
+n=$(nth_boot)
+case "${n}" in
+  1) 
+    require_writable_sysroot
+    # Hack this off for now
```

Comment:
Interesting to think about possibly using an Ignition config for this now as a test.

Reply:
Oh yes, more tests to be written and reworked now...just trying to port what exists.  This particular test though is more trying to test the "anaconda path" and not all the special stuff that happens w/ignition+ostree.